content/doc/developer/blueocean-plugin-development/index.adoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: developerchapter
+summary: How to develop plugins for Blue Ocean
+wip: true
+title: Extending Blue Ocean
+references:
+- url: https://github.com/jenkinsci/blueocean-plugin
+  title: Main Blue Ocean plugin repository
+- url: https://github.com/jenkinsci/jenkins-design-language
+  title: Design language repository with the styles and common components for Blue Ocean
+---
+
+The Blue Ocean UX plugin for Jenkins is designed to be extended by other plugins. 
+Like many other plugins, Blue Ocean has its own extension points. 
+
+Blue Ocean is a Single Page Application, with front end aspects contributed by many plugins, 
+and backend APIs contributed by many plugins as well. 
+
+The main difference for Blue Ocean is that front end aspects of the plugin use a lot of JavaScript, with JavaScript specific extension points. 
+If JavaScript makes you feel queasy than perhaps don't read further! 
```

Comment:
This queasy line should be dropped. Unless your alter ego is Brendan Eich, no need to be ashamed of JavaScript f or Blue Ocean :smiling_imp: